model_fitting/random_forest.py

```diff
@@ -97,7 +97,7 @@ def generate_heat_map(df, number_of_features, hits_data, number_of_samples, use_
     else:  # p-values data
         train_data = -np.log2(df)
 
-    cm = sns.clustermap(df, cmap="Blues", col_cluster=False, yticklabels=True)
+    cm = sns.clustermap(train_data, cmap="Blues", col_cluster=False, yticklabels=True)
```

Comment:
Need to remove (or at least comment) the conditions regarding tf-idf and p-values.
These values are already normalized, the current change will also reverse the colors in tf-idf and p-values.

Also note that controlled shuffles is considered as tf-idf (we should change that in future versions)

Please change the conditions to a simples single line:
train_data = np.log2(df+1) if hits_data else df